pkgs/servers/http/pomerium/default.nix

```diff
@@ -68,8 +90,11 @@ buildGoModule rec {
     //go:embed envoy.version
     var rawVersion string
     EOF
-    sha256sum '${envoy}/bin/envoy' > internal/envoy/files/envoy.sha256
-    echo '${envoy.version}' > internal/envoy/files/envoy.version
+    sha256sum '${envoy}/bin/envoy' > pkg/envoy/files/envoy.sha256
+    echo '${envoy.version}' > pkg/envoy/files/envoy.version
+
+    # put the built UI files where they will be picked up as part of binary build
+    cp ${ui}/libexec/pomerium/deps/pomerium/dist/* ui/dist
```

Comment:
Could we symlink this?

Reply:
Does not seem like it -- when trying with symlink instead of copy go throws this:
```
error: builder for '/nix/store/nvgagk6nf868ynlaqpxjid2hqcn31ccw-pomerium-0.18.0.drv' failed with exit code 1;
       last 10 log lines:
       > unpacking sources
       > unpacking source archive /nix/store/p2bcliccaw5f1vd0iraw4qswgq1nc7b6-source
       > source root is source
       > patching sources
       > applying patch /nix/store/ih5l43l8z9xwavckpv58f4yrg1pgpfsm-external-envoy.diff
       > patching file pkg/envoy/envoy.go
       > configuring
       > building
       > Building subPackage ./cmd/pomerium
       > ui/embed_local.go:14:13: pattern dist/*: cannot embed irregular file dist/index.css